docs/requirements.txt

```diff
@@ -1,20 +1,13 @@
-git+https://github.com/SymbiFlow/sphinx_symbiflow_theme.git#egg=sphinx-symbiflow-theme
```

Comment:
Seems to be a lot of unrelated changes to this file?

Reply:
Fixed!